docs/troubleshoot/upgrade-zowe-no-downtime.md

```diff
@@ -0,0 +1,91 @@
+# Upgrade Zowe in High availability
+
+As a systems programmer, you will need to upgrade Zowe regularly, to make sure you are using the latest features and to make sure you are using the latest security patches.
+
+This article describes the requirements and procedure to perform an upgrade of a Zowe installation in high availability mode, avoiding downtime.
+
+## Assumptions
+
+The following are assumptions about the installation prior to starting the upgrade process. Other settings are supported with adapted instructions.
+
+- Zowe is installed in a single shared zFS.
+- Zowe is setup in high availability mode with a sysplex distributor and two instances (A and B)
+- The upgraded instance will replace one of the existing instances (for the purposes of this procedure this will be instance B)
+- Instance A and B runs Zowe 2.4.0 and the upgraded instance will run a newer minor version (i.e. 2.10.0)
+
```

Comment:
What I want to point out here is the fact that the procedure can work for non-breaking changes. It's not certain that it will work for major version upgrades with potentially breaking changes.

Reply:
Also, the procedure is about upgrading one of the instances, to upgrade the second one the user should perform the same steps on the other instance.

Reply:
The main reason is that it doesn't feel like an assumption but more the reality afterwards.

Reply:
ok yes, I agree. I will move this to the introduction part of the article.